Impuls card

robingundlach opened the pull request:
Added Card

robingundlach committed:
Add Impulse Card

robingundlach committed:
Insert Card Cyan

robingundlach committed:
Added Title

stritti committed:
add package-lock for required 'npm ci'

robingundlach committed:
Added Impulse Card

Rearange margin and paddings

stritti commented on src/components/cards/ImpulseCard.vue:
```diff
@@ -0,0 +1,45 @@
+</script>
+<style lang="scss" scoped>
+.card-background{
+width: 100%;}
```
besser formatieren (Einrückungen)

enricofranke commented on src/components/cards/ImpulseCard.vue:
```diff
@@ -0,0 +1,45 @@
+   <div class="card-body">
+       <div class="card-content">
+           <div class="card-text"><p>Fahrrad fahren ist gut für die Umwelt.</p></div>
+           <div class="card-text card-text-space"><p>Da man kein CO<sub>2</sub> außstöst. Das ist mehr Text um zu zeigen das hier dann alles auf die SChnauze fliegen wird. lül.</p></div>
```
Typo bei SChnauze

enricofranke commented on src/components/cards/ImpulseCard.vue:
```diff
@@ -0,0 +1,45 @@
+       </div>
+   </div>
+</template>
+<script>
```
Wird Script benötigt, wenn nicht benutzt?

robingundlach committed:
Fix Comments

Commit:
Add Link to Home.vue (#3)

* Add Link to Home.vue

Add Signup, add login

* add package-lock for required 'npm ci'

* Update-home.vue

add change link to rounter-link

* Update-home-vue

changed the link to a router link so that the link searches by name.

* Update-home-vue

fix the problem

Co-authored-by: Stephan Strittmatter <stephan.strittmatter@sybit.de>

chrisWolter committed:
Show Card on StartPage

chrisWolter committed:
Merge remote-tracking branch 'origin/develop' into impuls-card

chrisWolter committed:
Add Semicolon
